pages/services/data-science-engine/1.0.0/install/index.md

```diff
@@ -0,0 +1,402 @@
+
+```json
+{
+ "service": {
+   "user": "<user>"
+ }
+}
+```
+
+{{ model.packageName }} runs all of its components in Docker containers. Since the Docker image contains a full Linux userspace with its own `/etc/users` file, it is possible for the user `nobody` to have a different UID inside the container than on the host system. Although user `nobody` has UID 65534 by convention on many systems, this is not always the case. As Mesos does not perform UID mapping between Linux user namespaces for a Docker containerizer, specifying a service user of `nobody` in this case will cause access failures when the container user attempts to open or execute a filesystem resource owned by a user with a different UID, preventing the service from launching. If the hosts in your cluster have a UID for `nobody` other than 65534, you will need to a provide valid UID for `nobody` in the configuration to run the service successfully. For example, on RHEL/Centos based distributions:
+
+```json
+{
+ "service": {
+   "user": "nobody",
+   "docker_user": "99"
+ }
+}
+```
+
+# Configure {{ model.packageName }} virtual network
+
+{{ model.techName }} can be launched in a virtual network and configured with network labels.
+Here's an example of a {{ model.packageName }} configuration for DC/OS overlay network:
+
+```json
+{
+ "service": {
+   "virtual_network_enabled": true,
+   "virtual_network_name": "dcos",
+   "virtual_network_plugin_labels": [
+       {"key": "key_1", "value": "value_1"},
+       {"key": "key_2", "value": "value_2"}
+   ]
+ }
+}
+```
+
+When {{ model.packageName }} is deployed in a virtual network, all submitted jobs will run in the same network until another network is specified in job submit arguments.
+You can check the existing limitations of virtual network support here.
+Minimal installation
+For development purposes, use dcos-vagrant to install {{ model.packageName }} on a local DC/OS cluster.
+
+1. Install a minimal DC/OS Vagrant according to the instructions [here][16].
+1. Install {{ model.packageName }}:
+    ```bash
+    dcos package install {{ model.packageName }}
+    ```
+1. Run a simple job:
+
+    ```bash
+    dcos {{ model.packageName }} run --submit-args="--class org.apache. {{ model.packageName }}.examples.{{ model.packageName }}Pi https://downloads.mesosphere.com/ {{ model.packageName }}/assets/ {{ model.packageName }}-examples_2.11-2.3.2.jar 30"
+    ```
+
+<p class="message--important"><strong>IMPORTANT:  </strong>A limited resource environment such as DC/OS Vagrant restricts some of the features available in {{ model.techName }}. For example, you must have enough resources to start up a 5-agent cluster, otherwise you will not be able to install DC/OS HDFS and enable the history server.</p>
+
+Also, a limited resource environment can restrict how you size your executors, for example with `{{ model.packageName }}.executor.memory`.
+
+# Multiple installations
```

Comment:
This is capped by the number of licenses a customer has paid for. So we can delete mentions of `dispatchers` but overall users can deploy `n` number of instances based on license count.

Reply:
Modified.